kapitan/inputs/copy.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019 The Kapitan Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import shutil
+
+from kapitan.inputs.base import InputType, CompiledFile
+from kapitan.resources import inventory
+
+logger = logging.getLogger(__name__)
+
+
+class Copy(InputType):
+    def __init__(self, compile_path, search_paths, ref_controller):
+        super().__init__("copy", compile_path, search_paths, ref_controller)
+
+    def compile_file(self, file_path, compile_path, ext_vars, **kwargs):
+        """
+        Write items in path as plain rendered files to compile_path.
+        path can be either a file or directory.
+        kwargs:
+            reveal: default False, set to reveal refs on compile
+            target_name: default None, set to current target being compiled
+        """
+        reveal = kwargs.get('reveal', False)
```

Comment:
This reveal and target_name aren't actually used anywhere in this function.
We can probably remove them from here (and the docstring above) to simplify the code?

Reply:
Removed them from the code and the docstring.